src/DependencyInjection/Configuration.php

```diff
@@ -69,6 +69,9 @@ private function addConverterSection(ArrayNodeDefinition $rootNode): void
                                         ->scalarNode('default')
                                             ->defaultValue(null)
                                         ->end()
+                                        ->scalarNode('nullsafe')
+                                            ->defaultValue(false)
+                                        ->end()
```

Comment:
This should be a `booleanNode`

```suggestion
                                        ->booleanNode('nullsafe')
                                            ->defaultFalse()
                                        ->end()
```

Reply:
Also: should we call it `skip_null` instead? Because that's more like what it does...

Reply:
I'm fine with that.

Reply:
```
private bool $skip_null = false,
```
I hate this snake cases, but ok, there are no getters and setters.

Reply:
I meant that only the config key should be snake case (because Symfony does it that way). 
The parameters/properties/variables should be written in camel case like everywhere else.

Reply:
Done.